docs/src/atomistic/reference/models/export.rst

```diff
@@ -16,3 +16,4 @@ define some information about the model, using :py:class:`ModelMetadata` and
 
 .. autoclass:: metatensor.torch.atomistic.MetatensorAtomisticModel
     :members:
+    :exclude-members: wrapped_module
```

Comment:
```suggestion
```

And did you check that `module` appears in the docs?

Reply:
No, and my way of including it also seems to break torchscript

Reply:
instead of `ModelInterface`, you can try to annotate with `torch.nn.Module`

Reply:
This seems to work, could you add a test that we can access the module after a full export/save/load cycle? And remove the line above in the docs as well as the `wrappedmodule` function?